multus/multus.go

```diff
@@ -221,7 +242,7 @@ func cmdDel(args *skel.CmdArgs, exec invoke.Exec, kubeClient k8s.KubeClient) err
 	}
 
 	if in.Kubeconfig != "" {
-		delegates, err := k8s.GetK8sNetwork(args, in.Kubeconfig, kubeClient, in.ConfDir)
+		delegates, _, err := k8s.GetK8sNetwork(args, in.Kubeconfig, kubeClient, in.ConfDir)
```

Comment:
We should probably remove the status annotation here, no?  The pod object may still be alive and in the API, but the annotation should be removed because the networks won't apply.